builder/profitbricks/builder_test.go

```diff
@@ -0,0 +1,57 @@
+package profitbricks
+
+import (
+	"testing"
+	"github.com/mitchellh/packer/packer"
+	"fmt"
+)
+
+func testConfig() map[string]interface{} {
+	return map[string]interface{}{
+		"image": "Ubuntu-16.04",
+		"password": "password",
+		"username": "username",
```

Comment:
`testBuilderAccBasic` uses `pbusername` and `pbpassword` but this test uses `username` and `password`. Which is correct? Are these different credentials? If so please add them to the docs.